index.html

```diff
@@ -81,6 +81,176 @@ <h2>Glad to see you!</h2>
+              <img src="./images/avail.svg" alt="Availability" />
+              <h2>Multi-Post Stories</h2>
+              <div class="card-caption">
+                <span>Canopy</span>
+                <img src="./images/Counter.svg" alt="counter" />
+                <span>Back End Dev</span>
+                <img src="./images/Counter.svg" alt="counter icon" />
+                <span>2015</span>
+              </div>
+              <div class="work-text">
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+              </div>
+              <div class="stacks">
+                <ul>
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                </ul>
+              </div>
+              <a href="#" class="work-btn">See Project</a>
+            </div>
+          </div>
+          <div class="card card-item-3">
+            <div class="card-work">
+              <img src="./images/gymfit.svg" alt="gymfit" />
+              <h2>Tonic</h2>
+              <div class="card-caption">
+                <span>Canopy</span>
+                <img src="./images/Counter.svg" alt="counter" />
+                <span>Back End Dev</span>
+                <img src="./images/Counter.svg" alt="counter icon" />
+                <span>2015</span>
+              </div>
+              <div class="work-text">
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+              </div>
+              <div class="stacks">
+                <ul>
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                </ul>
+              </div>
+              <a href="#" class="work-btn">See Project</a>
+            </div>
+          </div>
+          <div class="card card-item-4">
+            <div class="card-work">
+              <img src="./images/art-printing.svg" alt="Art printing" />
+              <h2>Multi-Post Stories</h2>
+              <div class="card-caption">
+                <span>Canopy</span>
+                <img src="./images/Counter.svg" alt="counter" />
+                <span>Back End Dev</span>
+                <img src="./images/Counter.svg" alt="counter icon" />
+                <span>2015</span>
+              </div>
+              <div class="work-text">
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+              </div>
+              <div class="stacks">
+                <ul>
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                </ul>
+              </div>
+              <a href="#" class="work-btn">See Project</a>
```

Comment:
- [ ] Kindly check your CSS file to apply the correct hover for the buttons. Your CSS does not load here for inline comment on the CSS file for some reason which I cannot figure out. Check the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) for the hover properties. See screenshot below. 
![image](https://user-images.githubusercontent.com/62849872/143620706-86ac321e-41dc-486f-aceb-500217c3a346.png)

Reply:
Hi, @t-yanick Kindly expatiate on what you mean by "Your CSS does not load here for inline comment on the CSS file" I don't quite understand. Thanks

Reply:
Hi @netman5 what I mean is simply that in the *Files changed* tab, your CSS did not load there. If it did I would have made the review comment on the css rather than the html. But it is not a problem, so worry not about it.

Reply:
Hi @t-yanick thanks for the quick response, honestly don't why it doesn't load to be honest, hey but thanks for the explanation. Effect the changes waiting for it to be reviewed.